doc/stream.tex

```diff
@@ -26,8 +26,8 @@
 
 Streams facilitate composition of higher-order functions like map, filter, and fold, while
 minimizing allocation of intermediate data structures. The \code{s/>} procedure allows
-expressions analogous to those using found in \csharp, Ruby, and other popular languages,
-in which a stream of values is passed through a "pipeline" of operators.
+expressions analogous to those found in \csharp, Ruby, and other popular languages, in
+which a stream of values is passed through a "pipeline" of operators.
```

Comment:
We should use \`\` and \'\' for the double-quotes, or switch to \\emph, or simply remove them since we're not really quoting anything here.

Reply:
Removed